collisions.qmd

```diff
@@ -4,15 +4,15 @@ title: "STATS19"
 
 <!-- https://a58improvements.commonplace.is/ -->
 
-# Introduction
+## Introduction
```

Comment:
Why is this a subheading not a heading? Will merge in any case but post-merge reply welcome.